Fhi.HelseId.Integration.Tests/TestFramework/NHNTTT/Dtos/DPoPProofParameters.cs

```diff
@@ -8,6 +8,8 @@ internal record DPoPProofParameters
 
         public string? PrivateKeyForProofCreation { get; set; }
 
-        public InvalidDPoPProofParameters? InvalidDPoPProofParameters { get; set; }
+        //// Removed the implementation as it was wrong. This is a property in TTT that
```

Comment:
Nit: Normal comments can have //
Commented out code should have //// if it was intentional / has meaning

Reply:
Fikser det i neste.